Meta/gn/README.md

```diff
@@ -6,7 +6,32 @@ GN is a metabuild system. It always creates ninja files, but it can create some
 
 This is a good [overview of GN](https://docs.google.com/presentation/d/15Zwb53JcncHfEwHpnG_PoIbbzQ3GQi_cpujYwbpcbZo/edit#slide=id.g119d702868_0_12).
 
-For more information, motivation, philosophy, and inspriation, see the LLVM documentation on its [GN build](https://github.com/llvm/llvm-project/tree/main/llvm/utils/gn#quick-start)
+For more information, motivation, philosophy, and inspiration, see the LLVM documentation on its [GN build](https://github.com/llvm/llvm-project/tree/main/llvm/utils/gn#quick-start)
+
+# Creating a gn build
+
+To create a GN build, you need to have GN installed. You can install it via homebrew on macOS, or via your package manager on Linux.
```

Comment:
Do we want to advertise BuildGN.sh? I found it pretty convenient over figuring out what package managers have what version of GN